components/slider/stories/slider.stories.js

```diff
@@ -205,9 +252,28 @@ Gradient.args = {
 	},
 	label: "Slider label that is long and wraps to the next line",
 };
+Gradient.tags = ["autodocs", "!dev"];
 
 export const SideLabel = Template.bind({});
```

Comment:
Sure, no I like that. I didn't change "Side Label" since that's just what it was called before, but your reasoning makes more sense (and that same story is called "With side label" on the docs site so...). 

I went with the "With" to keep all of those stories sort of together.

 It's in this commit, along with the "Sizing" name change, the without label story (formerly with label that was the same as default): https://github.com/adobe/spectrum-css/pull/2900/commits/4eaf516bcdf2ad8261f1a91307fa099f1af6ad22